### ISSUE TITLE
Koa support on the way!

### ISSUE THREAD

**Issue by gombosg** [slanatech/swagger-stats#67]
Hi, i have [forked](https://github.com/gombosg/swagger-stats) the repo and made a commit that enables swagger-stats to be used with Koa (or other frameworks), by simply substituting Express-specific calls with native Node equivalents. I tried to keep things simple.

I'm not making a PR yet. Give me a few days to test it in our app, I'm integrating Prometheus, Grafana and swagger-stats with our app. Just to make sure everything is working fine!

In the meantime you can test it.

The whole app could be made framework-agnostic by integrating this:
```
        appCtx.req.originalUrl = appCtx.request.originalUrl;
        appCtx.res.req = appCtx.req;
        appCtx.req.query = qs.parse(appCtx.request.query);
```
into the main middleware code (problem is that modifying req and res may not be a nice move).

**Comment by gombosg** [comment]
Take a look at the changes I made. It's working. I tried to decouple from Express/Koa, so the library now does its own query string parsing with `qs` (it's not standards compliant because of object fields, and Koa doesn't do it either).
Where it still uses the server is `req.body` and `req.params` - we probably don't want to include a router or body parser anyway, and this is optional stuff.
I also updated the readme, now it runs on Koa with 1 line.
